pom.xml

```
@@ -116,10 +116,12 @@
         </pluginRepository>
     </pluginRepositories>
 
-    <properties>
-        <jenkins.version>2.277.1</jenkins.version>
+    <properties>
+        <revision>2.9.4</revision>
+        <changelist>-SNAPSHOT</changelist>
+        <gitHubRepo>jenkinsci/startup-trigger-plugin</gitHubRepo>
+        <jenkins.version>2.401.3</jenkins.version>

```

I think that the `jenkins.version` and the `bom` version should match.
```suggestion
        <jenkins.version>2.387.3</jenkins.version>

```